src/parser/mage/fire/modules/features/CombustionSpellUsage.js

```diff
@@ -47,7 +69,7 @@ class CombustionSpellUsage extends Analyzer {
   suggestions(when) {
     when(this.suggestionThresholds)
       .addSuggestion((suggest, actual, recommended) => {
-        return suggest(<>You cast <SpellLink id={SPELLS.FIREBALL.id} /> or <SpellLink id={SPELLS.SCORCH.id} /> {this.castedWithInstants} times ({this.castsWithInstantsPerCombustion.toFixed(2)} per Combustion) while you had charges of <SpellLink id={SPELLS.FIRE_BLAST.id} /> or <SpellLink id={SPELLS.PHOENIX_FLAMES_TALENT.id} /> available. Make sure you are using up all of your charges of Fire Blast and Phoenix Flames before using Fireball or Scorch during Combustion.</>)
+        return suggest(<>You started to cast <SpellLink id={SPELLS.FIREBALL.id} /> {!this.hasBlasterMaster ? <> or <SpellLink id={SPELLS.SCORCH.id} /></> : '' } {this.badCastsStarted} times ({this.castsWithInstantsPerCombustion.toFixed(2)} per Combustion), and completed {this.badCastsCompleted} casts, while you had charges of <SpellLink id={SPELLS.FIRE_BLAST.id} /> {this.hasPhoenixFlames ? <> or <SpellLink id={SPELLS.PHOENIX_FLAMES_TALENT.id} /> </> : '' } available. Make sure you are using up all of your charges of Fire Blast {this.hasPhoenixFlames ? ' and Phoenix Flames' : '' } before using Fireball {!this.hasBlasterMaster ? 'or Scorch' : '' } during Combustion.</>)
```

Comment:
Some unneeded fragments (<>) in here, also maybe it would be neater to have it spread out on multiple lines ?

Reply:
The spell link inside the ternary won't work without the extra fragment. That's why it's there